stream-chat-android-ui-common/src/main/kotlin/com/getstream/sdk/chat/viewmodel/messages/MessageListViewModel.kt

```diff
@@ -194,12 +195,14 @@ public class MessageListViewModel @JvmOverloads constructor(
                 client.muteUser(event.user.id).enqueue()
             }
             is Event.BlockUser -> {
+                validateCid(cid)
+                val (channelType, channelId) = event.cid.split(':')
```

Comment:
I think we can use `client.channel(cid)` to validate cid and obtain `ChannelClient` which methods don't require channel type or id and we won't have to make `validateCid` public